cmd/tempo/app/app.go

```diff
@@ -96,21 +94,11 @@ func New(cfg Config) (*App, error) {
 		statFeatureEnabledAuth.Set(1)
 	}
 
-	statFeatureEnabledMetricsGenerator.Set(0)
-	if cfg.MetricsGeneratorEnabled {
-		statFeatureEnabledMetricsGenerator.Set(1)
-	}
-
 	statFeatureEnabledMultitenancy.Set(0)
-	if cfg.SearchEnabled {
+	if cfg.MultitenancyEnabled {
```

Comment:
Was this a bug we never noticed? It's a weird change

Reply:
yup, if you had search enabled then it raised the "mutlitenancy enabled" metric